torch/tensor.py

```diff
@@ -873,6 +873,46 @@ def rename(self, *names, **rename_map):
         # See Note [rename_ / rename API]
         return update_names(self, names, rename_map, inplace=False)
 
+    def to_sparse_csr(self, fill_value=0):
```

Comment:
I suggest removing the `fill_value` support from this PR as supporting fill value, in general, has not been decided.

Reply:
In that case only 0 can be interpreted as an 'empty' value. Would that be okay?

Reply:
yes, semantics should be the same as `to_sparse`.